docs/example/basic.rst

```diff
@@ -114,6 +114,33 @@ configuration:
 
 .. _virtualenv: https://pypi.org/project/virtualenv
 
+.. _script-command:
+
+Running scripts as commands
+----------------------------------------------
+
+Shell and batch scripts can be run like other commands with one caveat:
```

Comment:
This should be added under the commands configuration, not here 👍

Reply:
Just to make sure I understood correctly, the whole text should be in the `changedir` conf section?

Reply:
No, it should be under https://tox.readthedocs.io/en/latest/config.html#conf-commands

Reply:
Alright, I see it now! I'll be available to work on this later today.

Reply:
It is now in the commands section!